bridge-stp.in

```diff
@@ -212,7 +212,7 @@ case "$action" in
             # Add bridge to mstpd and configure.
             echo "Adding/configuring bridge '$bridge' ..."
             "$mstpctl" addbridge "$bridge" || continue
-            if [ -n "$config_cmd" ]; then
+            if [ -x "$config_cmd" ]; then
```

Comment:
adding `-x` here would mean that `$config_cmd` has to be an absolute path [ or a relative path that is accessible ]
which happens atm ; from what i can tell, the `$config_cmd` should be an absolute path

tho, if `$config_cmd` is not executable here, it should be fixed at build time i guess
will think about it a bit